mne/viz/_3d.py

```diff
@@ -1132,44 +1110,33 @@ def plot_alignment(info, trans=None, subject=None, subjects_dir=None,
     for data, color, alpha, scale in zip(datas, colors, alphas, scales):
         if len(data) > 0:
             with warnings.catch_warnings(record=True):  # traits
-                points = mlab.points3d(data[:, 0], data[:, 1], data[:, 2],
-                                       color=color, scale_factor=scale,
-                                       opacity=alpha, figure=fig)
-                points.actor.property.backface_culling = True
+                renderer.add_spheres(data, color, scale, alpha, True)
     if len(eegp_loc) > 0:
         with warnings.catch_warnings(record=True):  # traits
-            quiv = mlab.quiver3d(
+            renderer.add_3d_arrows(
```

Comment:
if 3d arrows is a vispy naming convention why not but if not I would suggest to keep names that are standard for vtk/mayavi. So quiver3d

Reply:
ok, I'll modify it accordingly